plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestJsonCassandraHandles.java

```diff
@@ -37,20 +37,24 @@
 public class TestJsonCassandraHandles
 {
     private static final Map<String, Object> TABLE_HANDLE_AS_MAP = ImmutableMap.of(
```

Comment:
We need a test for `CassandraQueryRelationHandle` right ?

Reply:
It is always nice to have a roundtrip test right ?

Reply:
`TestCassandraTableHandle` already exists. I will add another method there.